js/bcex.js

```diff
@@ -385,6 +591,11 @@ module.exports = class bcex extends Exchange {
         return this.fetchOrdersByType ('open', symbol, since, limit, params);
     }
 
+    async fetchClosedOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        let orders = await this.fetchOrders (symbol, since, limit, params);
+        return this.filterBy (orders, 'status', 'closed');
```

Comment:
This particular method has a conflict with pagination, the limit of closed orders won't match the limit of all orders, unfortunately. Therefore I switched the value of `has['fetchClosedOrders'] = 'emulated'`. This is explained in detail here: https://github.com/ccxt/ccxt/issues/1896#issuecomment-419090123